plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/AppEngineDeployPreferencesPanel.java

```diff
@@ -502,19 +502,13 @@ private Composite createBucketSection(Composite parent) {
   @VisibleForTesting
   public Job latestGcpProjectQueryJob;  // Must be updated/accessed in the UI context.
 
-  private Predicate<Job> isLatestQueryJob = new Predicate<Job>() {
-    @Override
-    public boolean apply(Job job) {
-      return job == latestGcpProjectQueryJob;
-    }
-  };
-
   private void refreshProjectsForSelectedCredential() {
     projectSelector.setProjects(Collections.<GcpProject>emptyList());
     latestGcpProjectQueryJob = null;
 
     Credential selectedCredential = accountSelector.getSelectedCredential();
     if (selectedCredential != null) {
+      Predicate<Job> isLatestQueryJob = job -> job == latestGcpProjectQueryJob;
```

Comment:
It might be easier to inline this and use `latestGcpProjectQueryJob::equals` — `Job` doesn't override equals anyways.  Or alternatively, `Predicate.isEqual(latestGcpProjectQueryJob)` returns a predicate, useful in case `latestGcpProjectQueryJob` may be `null`.